### DISCUSSION TITLE
Sensitivity warning on RateOptimization.pbix

### DISCUSSION THREAD

**Discussion by lundejd** [microsoft/finops-toolkit#1041] [Q&A] [answered]
I can't open this file, it says "Your organization has protected the file with a sensitivity label..."

I see there is a step in the release to remove this, maybe someone forgot?

**Comment by flanakin** [comment, marked as answer]
*sigh* Thanks for reporting. This is very manual. Looking forward to more automation options from the PBI team. The Rate optimization report has been updated. Please download the latest ([PBIX](https://github.com/microsoft/finops-toolkit/releases/latest/download/RateOptimization.pbix) / [PBIT](https://github.com/microsoft/finops-toolkit/releases/latest/download/RateOptimization.pbit)). Sorry about the inconvenience!
